test/action.yaml

```diff
@@ -7,12 +7,45 @@ inputs:
     default: true
   github-token:
     required: true
-    description: "GitHub token passed to coveralls. Usually - 'secrets.GITHUB_TOKEN'. Coveralls uses this token to verify the posted coverage data on the repo and create a new check based on the results. It is built into Github Actions and does not need to be manually specified in your secrets store."
+    description: "GitHub Personal Access Token that can checkout the consumer git repository. e.g. 'secrets.GITHUB_TOKEN'"
+  tf-cli-credentials-token:
+    required: true
+    description: "The terraform cli config credentials token"
+  tf-cli-config-file:
```

Comment:
I'd probably just call these `terraform-cli` rather than `tf-cli`

Reply:
This has been addressed.